modules/ROOT/pages/visibility.adoc

```diff
@@ -0,0 +1,122 @@
+.. Remove the existing *ACM Community User* profile.
+. Remove all profiles from your community sharing set:
+.. Open *Setup* > *Communities Settings*.
+.. In the *Sharing Sets* section, edit your community sharing set and remove all profiles from the *Selected Profiles*.
+.. If the *Selected Objects* in the sharing set include *CommunityApi*, remove *CommunityApi* from the *Selected Objects*.
+. If your community already has users, open *Setup* > *Users* and update each user to change the *User License* to *Customer Community Plus* and select the new *ACM Community User Plus* profile.
+. In the API Community Manager control panel, set the visibility of each of your APIs to *Members Only*.
+
+=== Define Member Groups (Public Groups)
+
+Use member groups, also known as public groups, to group members and then make individual APIs visible to the group. Visibility rules, also known as sharing settings, determine the APIs visible to a member group.
+
+To create a member group:
+
+. Open *Setup* > *Public Groups*.
+. Create a *Public Group*.
+. Enter a name, such as `External Developers`.
+. Click *Save*.
+
+To add members to a member group:
+
+. Open *Setup* > *Public Groups*.
+. Edit the group.
+. In the *Search* dropdown, select *Customer Portal Users*.
+. Add the members by moving them from *Available Members* to *Selected Members*.
++
+*Available Members* shows only users with the new *ACM Community User Plus* profile. If the user you want to add is not in *Available Members*, open the user record and verify the profile.
+. Click *Save*.
+
+=== Define Visibility Rules (Sharing Settings) for APIs
+
+Use visibility rules, also known as sharing settings, to define a set of conditions that determine if an API is visible to a specific member group.
+
+To create a new sharing set:
+
+. Open *Setup* > *Sharing Settings*.
+. In the dropdown *Manage sharing settings for*, scroll down and select *CommunityApi*.
+. In the *Sharing Rules* section, click *New*.
+. Enter a name for the rule.
+. In the *Select your rule type* section, select *Based on Criteria*.
+. Define your desired criteria using the available fields on the API, such as *Name* and *Visibility*.
++
+Use the *Add Filter Logic* option to define more complex rules with and/or conditions.
+. Select the member group that sees the APIs matching these conditions.
+
+In this example, an internal developer group sees publicly visible APIs and some internal APIs:
+
+image::sharing-settings.png[Sharing Settings]
+
+== Add Members to a Member Group
+
+To add members to a member group:
+
+. Open *Setup* > *Public Groups*.
+. Edit the group.
+. In the *Search* dropdown, select *Customer Portal Users*.
+. Add the members by moving them from *Available Members* to *Selected Members*.
++
+*Available Members* shows only users with the new *ACM Community User Plus* profile. If the user you want to add is not in *Available Members*, open the user record and verify the profile.
```

Comment:
```suggestion
*Available Members* shows only users with the new profile. If the user you want to add is not in *Available Members*, open the user record and verify the profile.
```

Reply:
As there are multiple profiles with confusingly similar names, I preserved this, in order to be specific and reduce confusion.